jfxtras-controls/src/main/java/jfxtras/scene/control/LocalDateTimeTextField.java

```diff
@@ -187,6 +187,36 @@ public void set(DateTimeFormatter value) {
 	public void setText(String value) { textObjectProperty.set(value); }
 	public LocalDateTimeTextField withText(String value) { setText(value); return this; }
 	
+        

```

Comment:
So. Here is the thing; I know that the 10 year old java style guide dictates things like that fields should go at the top of the class, and how to order methods, but sometimes things are outdated. And the advantages of open source is that there is no corporate rules to dictate how to do it, you can do as you feel is best. 

So when you write your own controls and code, you are free to code as you like, but in my classes I want to maintain my style and formatting. That means I'm not willing to use 30 lines of code just to add scaffolding methods around a property. I've settled onto a 5 line block; property class + javadoc, property, getter, setter, wither. So if you are changing my code, I want to stick to this style. Hence I cannot approve this.